src/main/java/io/jenkins/plugins/github/checks/ChecksConclusion.java

```diff
@@ -1,12 +1,24 @@
 package io.jenkins.plugins.github.checks;
 
+import org.kohsuke.github.GHCheckRun.Conclusion;
+
 public enum ChecksConclusion {
-    NONE,
-    ACTION_REQUIRED,
-    SKIPPED,
-    CANCELED,
-    TIME_OUT,
-    FAILURE,
-    NEUTRAL,
-    SUCCESS
+    NONE(null),
+    ACTION_REQUIRED(Conclusion.ACTION_REQUIRED),
+    SKIPPED(Conclusion.CANCELLED), // TODO: need a PR for GitHub API to add that
+    CANCELED(Conclusion.CANCELLED),
+    TIME_OUT(Conclusion.TIMED_OUT),
+    FAILURE(Conclusion.FAILURE),
+    NEUTRAL(Conclusion.NEUTRAL),
+    SUCCESS(Conclusion.SUCCESS);
+
+    private final Conclusion CheckRunConclusion;
```

Comment:
shouldn't it be this for java naming standards?
```suggestion
    private final Conclusion checkRunConclusion;
```